lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -84,8 +85,19 @@ class CourseEnrollmentSerializer(serializers.ModelSerializer):
     course = CourseOverviewField(source="course_overview", read_only=True)
     certificate = serializers.SerializerMethodField()
 
+    @classmethod
+    def get_catalog_course_runs(cls, user, course_keys):
+        """
+        Returns catalog runs against courses if marketing site is enabled.
+        """
+        if settings.FEATURES.get('ENABLE_MKTG_SITE'):
```

Comment:
@rlucioni What if `CatalogIntegration` is enabled but marketing site is disabled for lms? We do not want to make unnecessary request to catalog for data.